src/Controller/Api/CommitteesController.php

```diff
@@ -2,19 +2,49 @@
 
 namespace AppBundle\Controller\Api;
 
+use AppBundle\Repository\AdherentRepository;
+use AppBundle\Repository\CommitteeRepository;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
 
 class CommitteesController extends Controller
 {
     /**
      * @Route("/committees", defaults={"_enable_campaign_silence"=true}, name="api_committees")
      * @Method("GET")
      */
-    public function getApprovedCommitteesAction()
+    public function getApprovedCommitteesAction(): Response
     {
         return new JsonResponse($this->get('app.api.committee_provider')->getApprovedCommittees());
     }
+
+    /**
+     * @Route("/committees/count-for-referent-area", name="app_committees_count_for_referent_area")
+     * @Method("GET")
+     * @Security("is_granted('ROLE_REFERENT')")
+     */
+    public function committeesAndSupervisorsCountForReferentManagedAreaAction(AdherentRepository $adherentRepository, CommitteeRepository $committeeRepository): Response
```

Comment:
what about `getCommitteeStatsAction` or `getCommitteesCountersAction` ?

Reply:
yes, it will be better to change the name of method https://github.com/EnMarche/en-marche.fr/pull/2403/files#diff-b9035c7930cd644293369885f92ba31cR36

Reply:
Ok, I change to `getCommitteeStatsAction`.